docs/examples/lipstick-pig.md

```diff
@@ -0,0 +1,315 @@
+In this example we'd like to discuss the effectiveness of debiasing techniques in word embeddings.
+This example is heavily inspired by the [lipstick on a pig paper](https://arxiv.org/pdf/1903.03862.pdf)
```

Comment:
It's better to be written as:

inspired by the \["Lipstick on a Pig"](https://arxiv.org/pdf/1903.03862.pdf) paper